File: Java/src/main/java/knowledge_layer/AdaptationGoalManager.java
package knowledge_layer;

import specification.AdaptationGoal;

public class AdaptationGoalManager {
    private AdaptationGoal goal;

    public AdaptationGoalManager(AdaptationGoal adaptationGoal) {
        goal = adaptationGoal;
    }

    public AdaptationGoal getGoal() {
        return goal;
    }

    public void setGoal(AdaptationGoal newGoal) {
        goal = newGoal;

    }
}

File: Python/knowledge_layer/AdaptationGoalManager.py
from specification.AdaptationGoal import AdaptationGoal


class AdaptationGoalManager:

    def __init__(self, adaptationGoal: AdaptationGoal):
        self._goal = adaptationGoal

    def getGoal(self) -> AdaptationGoal:
        return self._goal

    def setGoal(self, newGoal: AdaptationGoal):
        self._goal = newGoal
